services/_shared/personas/code_reviewer/verify.py

```
@@ -0,0 +1,182 @@
+        _rule_matches(finding.rule_name, _CODE_EXECUTION_MARKERS)
+        or _rule_matches(finding.message, _CODE_EXECUTION_MARKERS)
+    ):
+        return "non_code_file"
+
+    if source is None:
+        return None  # no evidence either way - keep
+
+    if not _is_prose_file(finding.file) and _rule_matches(
+        finding.rule_name, _ASYNC_FAMILY_MARKERS
+    ):
+        if _enclosing_chain_is_sync(source, finding.line) is True:
```

Ruff (0.15.21)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)